### DISCUSSION TITLE
Year when the device was manufactured/released

### DISCUSSION THREAD

**Discussion by dfranquesa** [linuxhw/HWInfo#6] [Q&A] [answered]
How do you get the Year when the device was manufactured/released? Thanks! David

**Comment by linuxhw** [comment]
It's minimal release date of the BIOS among all instances of the device model in the database.

**Comment by dfranquesa** [comment, marked as answer]
Smart, would it be possible to get a list of models and the minimal release date of the BIOS? 
If can be useful to improve, I have seen that some laptops also have a year of manufacture of the monitor. Maybe this date is lower and can be used as a manufacturer year candidate.

**Reply by linuxhw** [reply]
Good idea, need to check this.
